test/integration/default/serverspec/install_spec.rb

```diff
@@ -1,6 +1,17 @@
 require 'serverspec'
 
-describe file('/usr/local/bin/composer') do
+set :backend, :exec
+
+# these vars must correspond with attributes' values
+composer_install_path = '/usr/local/bin'
+composer_version = '1.4.1'
```

Comment:
Again, this should also be dependent on the attribute. Hardcoding this is a no-go